src/data-tutorials/users.tsx

```diff
@@ -961,6 +961,13 @@ const Users: User[] = [
     website: 'blog/tutorials/build-your-own-full-stack-dao-on-the-celo-blockchain',
     tags: ['celosage', 'solidity', 'celo', 'react', 'intermediate'],
   },
+  {
+    title: 'Build Your Own Full Stack NFT A.I Art Minter On The Celo Blockchain',
+    description: 'In this tutorial, we will build a full stack nft digital art minter that will generate our art with artificial intelligence.',
+    preview: require('./showcase/intermediate/build_your_own_full_stack_nft_a.i_art_minter_on_the_celo _blockchain.png'),
```

Comment:
This was creating issues here, but I changed it , make sure it does not happen in future